scripts/st2_bootstrap.sh

```diff
@@ -52,6 +53,10 @@ setup_args() {
           PASSWORD="${i#*=}"
           shift
           ;;
+          --force-branch=*)
```

Comment:
I think we're plastering this script with more unpopular features used just for a couple of times, but I'm OK if you really need it.

Just please include some basic description in comments, at least will help to know what is `--force-branch` means.

Reply:
I think it's a good idea to have this flag to make it possible to test with a non-master branch.

In fact, I believe we had it at some point, but dunno what happened to it.

Reply:
I agree comment is a good idea.

Added in f574f47418dc845a8b29d5dc30939b1757e5be39.